src/runtime/virtcontainers/remote.go

```diff
@@ -134,6 +134,10 @@ func (rh *remoteHypervisor) AttestVM(ctx context.Context) error {
 
 func (rh *remoteHypervisor) StopVM(ctx context.Context, waitOnly bool) error {
 
+	// waitOnly doesn't make sense for remote hypervisor and suited for local hypervisor.
+	// Instead use a similar logic like StartVM to handle StopVM with timeout.
+	timeout := int(rh.config.RemoteHypervisorTimeout)
```

Comment:
Does this need the same logic as `StartVM`:
```
	minTimeout := defaultMinTimeout
	if rh.config.RemoteHypervisorTimeout > 0 {
		minTimeout = int(rh.config.RemoteHypervisorTimeout)
	}
```

Reply:
I'll modify the code to use the defaultMinTimeout

Reply:
@bpradipt, if you're already modifying the code, please, also change "Ad" -> "Add" in the commit message.